packages/forma-36-react-components/src/components/CopyButton/CopyButton.stories.tsx

```diff
@@ -1,31 +1,33 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
-import { text, select } from '@storybook/addon-knobs';
+import type { Meta, Story } from '@storybook/react/types-6-0';
 
 import CopyButton from './CopyButton';
+import type { CopyButtonProps } from './CopyButton';
+import notes from './README.mdx';
 
-storiesOf('Components/CopyButton', module)
-  .addParameters({
+export default {
+  argTypes: {
+    children: { control: { disable: true } },
+    className: { control: { disable: true } },
+    testId: { control: { disable: true } },
+    tooltipText: { control: 'text' },
+    tooltipCopiedText: { control: 'text' },
```

Comment:
Do we need this? I thought storybook would automatically guess the controls for these props based on their types